docs/build-clio.md

```diff
@@ -141,3 +141,43 @@ If you wish to develop against a `rippled` instance running in standalone mode t
 
 1. Advance the `rippled` ledger to at least ledger 256.
 2. Wait 10 minutes before first starting Clio against this standalone node.
+
+## Building with a Custom `libxrpl`
+
+Sometimes, during development, you need to build against a custom version of `libxrpl`. (For example, you may be developing compatibility for a proposed amendment that is not yet merged to the main `rippled` codebase.) To build Clio with compatibility for a custom fork or branch of `rippled`, follow these steps:
+
+1. First, pull/clone the appropriate `rippled` fork and switch to the branch you want to build. For example:
+
+    ```sh
+    git clone https://github.com/XRPL-Labs/xrpld-hooks
+    cd xrpld-hooks
+    ```
+
+2. Export a custom package to your local Conan store using a user/channel:
+
+    ```sh
+    conan export . my/feature
+    conan upload --all xrpl/2.2.2-rc3@my/feature -r conan-non-prod
```

Comment:
We don't need this line, which will upload the package to our conan artifactory. I think we only need to export to local.